dnfdaemon-client/commands/repoquery/repoquery.cpp

```diff
@@ -96,13 +98,20 @@ class PackageDbus {
     explicit PackageDbus(const dnfdaemon::KeyValueMap & rawdata) : rawdata(rawdata){};
     int get_id() { return rawdata.at("id"); }
     std::string get_name() const { return rawdata.at("name"); }
-    unsigned long get_epoch() const { return rawdata.at("epoch"); }
+    std::string get_epoch() const { return std::to_string((int)rawdata.at("epoch")); }
```

Comment:
Please do the conversion to string directly in `dnfdaemon-server/services/rpm/rpm.cpp:package_to_map()` where the `rawdata` dictionary is created.
Also remove `(int)` c-style cast.

Reply:
Hm, it looks like `libdnf::rpm::Package.get_epoch()` has already been changed to return `std::string` so no conversion is needed.